bones/utils.go

```diff
@@ -1,6 +1,8 @@
 package bones
 
-import "google.golang.org/protobuf/reflect/protoreflect"
+import (
```

Comment:
Interesting - go fmt creates this as a single line import when there is only one import. If you add a second, it formats it with parens, as expected. But if you remove the need for that second import, it goes back to one import but leaves the parens. A little annoying for consistency, but oh well, not to matter.

Reply:
yup i stuck the const.go in this file first. sorry! i've manually removed the ().